### PR TITLE
add basic support for llama3.2

### PR DESCRIPTION
**Pull request by MeouSker77**
### 4. How to test?
- [ ] N/A
- [ ] Unit test: Please manually trigger the PR Validation [here](https://github.com/intel-analytics/ipex-llm-workflow/actions/workflows/llm-PR-validation.yml) by inputting the PR number (e.g., `1234`). And paste your action link here once it has been successfully finished.
- [ ] Application test
- [ ] Document test
- [ ] ...

### CODE REVIEW

**Comment by MeouSker77** [comment]
PR validation: <https://github.com/intel-analytics/ipex-llm-workflow/actions/runs/11047309055>

File: python/llm/src/ipex_llm/transformers/models/llama_new.py [hunk 1]
```diff
@@ -0,0 +1,236 @@
+#
```
Comment (jason-dai) [inline python/llm/src/ipex_llm/transformers/models/llama_new.py:1]: change to llama32.py?

File: python/llm/src/ipex_llm/transformers/kv.py [hunk 2]
```diff
@@ -24,19 +24,28 @@
...
-from typing import Optional, Dict, Tuple, Any
+from typing import Optional, Dict, Tuple, Any, List
 from transformers.cache_utils import DynamicCache
 from ipex_llm.utils.common.log4Error import invalidInputError
 
 
 class DynamicFp8Cache(DynamicCache):
+    def __init__(self, num_hidden_layers: Optional[int] = None) -> None:
+        super().__init__()
+        # fix transformers >= 4.45
```
Comment (jason-dai) [inline python/llm/src/ipex_llm/transformers/kv.py:35]: may also need to fix `DynamicCompressCache` and `DynamicCompressFp8Cache`

File: python/llm/src/ipex_llm/transformers/models/llama_new.py (hunk 1 above)
Comment (MeouSker77) [reply python/llm/src/ipex_llm/transformers/models/llama_new.py]: changed

File: python/llm/src/ipex_llm/transformers/kv.py (hunk 2 above)
Comment (MeouSker77) [reply python/llm/src/ipex_llm/transformers/kv.py]: fixed

**Review by Oscilloscope98 — APPROVED** [review]

**Pull request merged**